Search-algorithms/src/base/SelectionSort.java

```diff
@@ -0,0 +1,20 @@
+package base;
+
+public class SelectionSort {
+
+  static int i, j, min, tmp;
```

Comment:
Questi sono campi inutili che usi solo nella classe sorting, dovresti definirli solo all'interno del metodo in cui le usi

Reply:
fixato poi controlla